docs/yarn.md

```diff
@@ -0,0 +1,178 @@
+# yarn
+
+<https://v3.yarnpkg.com/>
+
+* Overview [in the README][readme-yarn]
+* [Cachi2's Yarn support scope](#cachi2s-yarn-support-scope)
+    * [Supported Yarn versions](#supported-yarn-versions)
+    * [Supported Yarn protocols/locators](#supported-yarn-protocolslocators)
+    * [Dealing with .yarnrc.yml](#dealing-with-yarnrcyml)
+    * [Dealing with Yarn Zero-Installs](#dealing-with-yarn-zero-installs)
+    * [Dealing with plugins](#dealing-with-plugins)
+* [Specifying packages to process](#specifying-packages-to-process)
+    * [Controlling Yarn's behavior](#controlling-yarns-behavior)
+    * [Downloading dependencies](#downloading-dependencies)
+* [Using fetched dependencies](#using-fetched-dependencies)
+    * [Known pitfalls](#known-pitfalls)
+    * [Building your project using the pre-fetched Yarn dependency
+cache](#building-your-project-using-the-pre-fetched-yarn-dependency-cache)
+
+## Cachi2's Yarn support scope
+
+### Supported Yarn versions
+Cachi2 only currently only supports Yarn in version 3.
+
+_Note: newer versions of Yarn are likely to be added in future releases._
+
+### Supported Yarn protocols/locators
+
+Cachi2 currently supports all standard
+[Yarn v3 protocols](<https://v3.yarnpkg.com/features/protocols/>) except for:
+- [Exec](https://v3.yarnpkg.com/features/protocols#exec)
+- [Git/GitHub](https://v3.yarnpkg.com/features/protocols#git)
+
+Due to the nature of how the two protocols above work, mainly related to potentially executing
+arbitrary code, adding support for them with future releases of Cachi2 is unlikely. For further
+details on Yarn protocols and their practical ``package.json`` examples, please head to the
+official Yarn documentation on protocols linked earlier in this section.
+
+### Dealing with .yarnrc.yml
+Cachi2 parses the project's ``.yarnrc.yml`` file and analyzes configuration settings. Before cachi2
+proceeds with the actual dependency fetching, it verifies whether all [configuration
+settings](https://v3.yarnpkg.com/configuration/yarnrc) that set a path to a resource don't point
+outside of the source repository, so in order to avoid any issues reported by Cachi2 in this regard
+make sure all your project resource references are bound by the repository. Part of the analysis of
+the repository's ``.yarnrc.yml`` file is detection of plugin usage which is further explained in
+[Dealing with plugins](#dealing-with-plugins).
+
+### Dealing with Yarn Zero-Installs
+
+Yarn's [PnP Zero-Installs](https://v3.yarnpkg.com/features/zero-installs/) are unsupported due to
+the potentially unplugged dependencies checked into the repository which simply make it
```

Comment:
Wdyt about adding a link to clarify what [unplugged dependencies](https://v3.yarnpkg.com/advanced/lexicon#unplugged-package) are?